script.module.pyaes/addon.xml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyaes"
+    name="PyAES"
+    version="1.6.1"
+    provider-name="Bas Rieter">
```

Comment:
You aren't the maintainer of this library.

Reply:
Added the official maintainer here 'ricmoo'